### ISSUE TITLE
eslint --init failed when answered with "Inspect your JavaScript file"

### ISSUE THREAD

**Issue by yeonjuan** [eslint/eslint#13156]
**Tell us about your environment**

* **ESLint Version:** v7.0.0-alpha.3
* **Node Version:** v11.15.0
* **npm Version:** v6.7.0

**What parser (default, Babel-ESLint, etc.) are you using?**

**Please show your full configuration:**

<details>
<summary>Configuration</summary>

```js

```

</details>

**What did you do? Please include the actual source code causing the issue, as well as the command that you used to run ESLint.**

```bash
$ eslint --init

? How would you like to use ESLint? To check syntax, find problems, and enforce code style
? What type of modules does your project use? None of these
? Which framework does your project use? Vue.js   
? Does your project use TypeScript? No
? Where does your code run? Browser
? How would you like to define a style for your project? Inspect your JavaScript file(s)
? Which file(s), path(s), or glob(s) should be examined? test.js
? What format do you want your config file to be in? JavaScript
```

**What did you expect to happen?**

configuring eslint successfully

**What actually happened? Please include the actual, raw output from ESLint.**

configure failed

```bash
Failed to load plugin 'vue' declared in 'BaseConfig': Cannot find module 'eslint-plugin-vue'
Require stack:
- /Users/yeonjuan/Desktop/open-source/new-eslint-committer/test/__placeholder__.js
Referenced from: BaseConfig
```

**Are you willing to submit a pull request to fix this bug?**

Yes

I found this bug in PR #12738
When we answered as below (**Inspect your JavaScript file(s)** ), 
It seems ESlint initializer tries to lint the target file ( **test.js** )  to create a proper config.

```bash
? How would you like to define a style for your project? Inspect your JavaScript file(s)
? Which file(s), path(s), or glob(s) should be examined? test.js
```

But at that time, If some plugins(eg. react, vue, typescript-eslint) needed are not installed, the initializer failed. 

It's because the function `configureRules()`  is called before modules installed (`askInstallModules`)

* configureRules
https://github.com/eslint/eslint/blob/c41de382a330d4789488fd2dcf6db5a3163bb5d2/lib/init/config-initializer.js#L154

* askInstallModules
https://github.com/eslint/eslint/blob/c41de382a330d4789488fd2dcf6db5a3163bb5d2/lib/init/config-initializer.js#L617

**Comment by kaicataldo** [comment]
@yeonjuan Sorry we missed this. Is this still an issue? If so, can you reopen it and assign it to yourself?
